lti_consumer/tests/unit/test_utils.py

```diff
@@ -0,0 +1,53 @@
+"""
+Utility functions used within unit tests
+"""
+
+from webob import Request
+from mock import Mock
+
+from xblock.fields import ScopeIds
+from xblock.runtime import KvsFieldData, DictKeyValueStore
+
+from workbench.runtime import WorkbenchRuntime
+
+
+def make_xblock(xblock_name, xblock_cls, attributes):
```

Comment:
Nice!  I think we should add a version of this function (and the next one) to xblock-utils.